integration/e2emimir/services.go

```diff
@@ -212,6 +212,44 @@ func NewSingleBinary(name string, flags map[string]string, options ...Option) *M
 	)
 }
 
+func NewReadInstance(name string, flags map[string]string, options ...Option) *MimirService {
+	return newMimirServiceFromOptions(
+		name,
+		map[string]string{
+			"-target":    "read",
+			"-log.level": "warn",
+		},
+		flags,
+		options...,
+	)
+}
+
+func NewWriteInstance(name string, flags map[string]string, options ...Option) *MimirService {
+	return newMimirServiceFromOptions(
+		name,
+		map[string]string{
+			"-target":    "write",
+			"-log.level": "warn",
+			// Speed up startup.
+			"-ingester.ring.min-ready-duration": "0s",
```

Comment:
Other integration tests set this option for all ingesters (and single binary instances), but I'm unclear of the implications, apart from speeding up startup. Is this the right thing to do here?

Reply:
Not relevant for the test, but for my own understanding: what does this achieve in contrast to mechanisms like setting limits on the number of unavailable pods in Kubernetes' rollout configuration?